qiskit/tools/visualization/_state_visualization.py

```diff
@@ -18,6 +18,7 @@
 from matplotlib.patches import FancyArrowPatch
 from mpl_toolkits.mplot3d import proj3d
 import numpy as np
+from qiskit import QISKitError
```

Comment:
We did recently add a VisualizationError class, I'm wondering if that would be more useful here.

https://github.com/Qiskit/qiskit-terra/blob/master/qiskit/tools/visualization/_error.py#L13

What do you think?

Reply:
I didn't know about that one, I'll change it over.